docs/modules/ROOT/pages/how-to-guides/managing-environments/proc_creating_your_own_environment.adoc

```diff
@@ -1,31 +1,63 @@
-
-To create your own environment and upload your own `kubeconfig` file, complete the following steps in the {ProductName} UI: 
-
-. Go to *Environments* > *Create Environment*. 
-. Complete all the required fields for *Create Environment*.  
+* You must have credentials for full read and write access to the namespace for your cluster.   
+
+.Procedure
+
+To create your own environment, you first need to create a functional `kubeconfig` file. Then, using that file, you add the cluster as a new environment in the {ProductName} UI.
+
+. In your preferred text editor, create a `kubeconfig` file based on the following example:
+
++
+
+[source]
+--
+apiVersion: v1
+clusters:
+- cluster:
+    server: https://api.sandbox-m2.ll9k.p1.openshiftapps.com:6443
+  name: api-sandbox-m2-ll9k-p1-openshiftapps-com:6443
+contexts:
+- context:
+    cluster: api-sandbox-m2-ll9k-p1-openshiftapps-com:6443
+    namespace: rhn-support-csears-dev
+    user: rhn-support-csears/api-sandbox-m2-ll9k-p1-openshiftapps-com:6443
+  name: rhn-support-csears-dev/api-sandbox-m2-ll9k-p1-openshiftapps-com:6443/rhn-support-csears
+current-context: rhn-support-csears-dev/api-sandbox-m2-ll9k-p1-openshiftapps-com:6443/rhn-support-csears
+kind: Config
+preferences: {}
+users:
+- name: rhn-support-csears/api-sandbox-m2-ll9k-p1-openshiftapps-com:6443
+  user:
+    token: sha256~<remainder of token omitted>
+--
+
++
+
+. In the {ProductName} UI, go to *Environments* > *Create Environment*. 
+. Enter a name for the new environment.  
 . Complete all the required fields for *Cluster information*: 
 .. For *Select cluster*, select that you are bringing your own cluster.  
 .. For *Cluster type*, select *Non-OpenShift* or *OpenShift*. 
-. Upload your `kubeconfig` file.
-. Attach the namespace to the cluster that you selected for this application.
+.. Upload your `kubeconfig` file, or paste its contents into the space provided.
+.. Specify the namespace for the cluster that you are using.
+. Select *Create environment*.
 . Confirm that your environment is created and is accessible under *Environments*. 
 
 .Troubleshooting 
 
-If there were any errors while creating your environment, complete the following steps: 
+If any errors occur while creating your environment, complete the following steps: 
+
+. Make sure the token you provided in your `kubeconfig` is valid. If the token in your `kubeconfig` has expired, log in to your cluster and request a new token.
+. Verify that you entered the correct values in the correct places in your `kubeconfig` file. When reading the example file, you can easily mistake the namespace, `rhn-support-csears-dev`, for the username, `rhn-support-csears`, and vice versa.  
+. Confirm that your cluster can support another pod for deploying your application.
+. If you do not find any errors in your `kubeconfig` file, go to *Environments* in the {ProductName} UI, select the three dots next to your new environment, delete the failed environment, and create it again.
```

Comment:
Failed environment...How does a user confirm it?
Also, how about this:

```suggestion
. If you do not find any errors in your `kubeconfig` file, go to *Environments* in the {ProductName} UI, select the three dots next to the environment you created, select Delete environment, and then create a new environment.
```

Reply:
Made this change, but bolded the UI text and just wrote "Delete," since that's what's in the UI